docs/dev_101_states.md

```diff
@@ -37,6 +37,21 @@ def setup(hass, config):
     return True
 ```
 
+You'll also need to create the file `<config dir>/custom_components/hello_state/manifest.json` and copy the below example code.
+
+```python
+{
+    "domain": "hello_state",
+    "name": "Hello State",
+    "version": "0.1",
+    "documentation": "https://www.example.com",
+    "dependencies": [],
+    "codeowners": [],
+    "requirements": [],
+    "iot_class": "cloud_polling"
```

Comment:
```suggestion
    "iot_class": "cloud_polling"
```
This example integration does not use the cloud? `local_push` is probably more fitting here

Reply:
Agree, will update.  Wanted to be consistent across our documentation.